.github/workflows/ci-validation.yml

```diff
@@ -36,9 +36,16 @@ jobs:
     - name: Build (Ubuntu 20.04)
       run: |
         sudo cp -rf . /p4c/
-        (cd /p4c/ && sudo -E tools/ci-build.sh)
+        cd /p4c/ && sudo -E tools/ci-build.sh || echo "BUILD_SUCCESS=false" >> $GITHUB_ENV
```

Comment:
(cd /p4c/ && sudo -E tools/ci-build.sh && echo "BUILD_SUCCESS=true" >> $GITHUB_ENV)

Reply:
Will wait for #3792.